Mintfile

```diff
@@ -1,3 +1,3 @@
 nicklockwood/SwiftFormat@0.47.12
 SwiftGen/SwiftGen@6.5.1
-realm/SwiftLint@0.47.0
+realm/SwiftLint@0.51.0
```

Comment:
Updating the SwiftLint causes a lot of warnings because, by default, it introduces a lot of new rules. 

In order to follow the "This change follows zero ⚠️ policy (required)", we need to update all the codebase and remove those warnings in this PR in order to be approved 👍

Reply:
👌

Reply:
Hmm... the `attributes` rule already enabled seems to have become stricter.